rama-http-types/src/request.rs

```diff
@@ -0,0 +1,107 @@
+use crate::body::Body;
+use http::{Extensions, HeaderMap, HeaderValue, Method, Uri, Version, request::Parts};
+
+/// Type alias for [`http::Request`] whose body type
+/// defaults to [`Body`], the most common body type used with rama.
+pub type Request<T = Body> = http::Request<T>;
+
+/// [`HttpRequestParts`] is used in places where don't need the [`Body`] of the [`http::Request`]
```

Comment:
`Body` here should link to the http_body::Body from `crate::dep`

ALso try to link to `crate::Request` instead of `http::` versions, so we keep it in crate.